resources/views/admin/courses.blade.php

```diff
@@ -35,16 +35,15 @@
                                 <div class="col-lg-3 col-md-6 col-12 col-sm-6">
                                     <div class="blogThumb">
                                         <div class="thumb-center"><img class="img-responsive" alt="user"
-                                                                       src="/admin/assets/img/course/course4.jpg"></div>
+                                                                       src="/admin/assets/img/course/course_image.webp"></div>
                                         <div class="course-box">
-                                            <h4>{{ $course->courseName }}</h4>
-                                            <div class="text-muted"><span class="m-r-10">April 23</span>
-                                                <a class="course-likes m-l-10" href="#"><i
-                                                        class="fa fa-heart-o"></i> 654</a>
+{{--                                            <h4>{{ $course->courseName }}</h4>--}}
```

Comment:
These lines are commented out so doesn't look like we need them. Go ahead and remove the unneeded commented out code from the files. otherwise everything else looks fine.